addons/website/static/src/js/content/website_root.js

```diff
@@ -34,10 +34,15 @@ var WebsiteRoot = publicRootData.PublicRoot.extend(KeyboardNavigationMixin, {
         });
         return this._super(...arguments);
     },
+    destroy() {
```

Comment:
docString

Reply:
Also usually methods are in their call order => init, willStart, start, destroy

Reply:
> Also usually methods are in their call order => init, willStart, start, destroy (in website atleast)

Reply:
:+1: